dags/als-kp-dataset-dag.py

```diff
@@ -41,10 +43,7 @@
     "cpath_api_url": Param(
         "https://fair.dap.c-path.org/api/collections/als-kp/datasets", type="string"
     ),
-    "collection_name": Param("Dataset collection (Production)", type="string"),
```

Comment:
collection_name and collection_description were used to create a new dataset collection. But we are retrieving the existing dataset collection now, so these parameters are no longer needed.